tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "outDir": "./esm",
+    "sourceMap": true,
+    "noImplicitAny": false,
+    "module": "es6",
+    "moduleResolution": "node",
+    "target": "es2017",
```

Comment:
ES2015 plz

Reply:
done, but be aware typescript does not transpile, if advanced Features are needed, there needs to be a babel step